install/empty_data.php

```diff
@@ -4382,7 +4382,157 @@ public function getEmptyData(): array
                 'items_id'         => '37',
                 'type'             => '1',
                 'notifications_id' => '40',
-            ],
+            ], [
+                'id'               => '159',
+                'items_id'         => '1',
+                'type'             => '1',
+                'notifications_id' => '1',
```

Comment:
Not easy to review only IDs 😕
Can you add a test to verify that in `getEmptyData()`, all entries in `$tables['glpi_notifications']` have at least one recipient?

Reply:
A good test could also be to get all the existing notifications, get the corresponding class, call the `addNotificationTargets()` method and verify that the `Administrator` is one of the default targets when it is a valid target for this specific notification.